entrypoint.sh

```
@@ -2,6 +2,28 @@
+REGEX=".*\.($REGEX)$"
+
+# Determine clang-format flags based on --fix flag
+if [ "$FIX_FLAG" = "--fix" ]; then
+  FORMAT_FLAGS="-i"
+else
+  FORMAT_FLAGS="--dry-run -Werror"
+fi
+
+# Run clang-format on all matching files
+find "$SCANDIR" -regextype posix-extended -regex "$REGEX" -print0 \
+  | xargs -0 -P12 -n1 clang-format $FORMAT_FLAGS --style=$STYLE
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)